sigstore-public-deployment-spec.md

```diff
@@ -191,3 +191,40 @@ The Sigstore project provides the resources necessary to deploy private Sigstore
 * [sigstore/scaffolding](https://github.com/sigstore/scaffolding)
 * [sigstore/sigstore-probers](https://github.com/sigstore/sigstore-probers)
 * [sigstore/policy-controller](https://github.com/sigstore/policy-controller)
+
+### 5.2 Supported Algorithms
+
+The Sigstore public good instance supports a subset of the algorithms defined
+in the [Algorithm Registry](./algorithm-registry.md). Clients that interoperate
+with the public good instance **MUST** support
+these algorithms in their respective contexts.
+
+#### 5.2.1 TUF
+
+The public good instance uses `ecdsa-sha2-256-nistp256` for all TUF signing keys.
+
+#### 5.2.2 Fulcio
+
+The public good instance uses `ecdsa-sha2-384-nistp384` for Fulcio's
+certificate chain and `ecdsa-sha2-256-nistp256` for Fulcio's certificate
+transparency log.
```

Comment:
This is all I could find from the TUF distributed keys, but I remember someone mentioning that Fulcio has/had RSA certs as well. So someone should double-check this!

Reply:
The staging instance's CT log public key IIRC. We also need to mention the CT log service.

Reply:
It was the original CT log in staging - https://github.com/sigstore/root-signing-staging/blob/ddbbed5cc101919b8294d1df09298768c23353c2/targets/trusted_root.json#L47

I'm going to guess most clients don't support RSA currently

Reply:
> I'm going to guess most clients don't support RSA currently

I had to double-check, but we actually do still support this in sigstore-python! We have a `Keyring` abstraction that we use for both Rekor and the CT log keys, and that abstraction allows the standard ECC pairings + RSA pairings:

https://github.com/sigstore/sigstore-python/blob/2ff9ee999040b4f5e5231bc8b0b89d6b6a0d1f7f/sigstore/_internal/trust.py#L96-L107

(Not sure about other clients, however.)